classes/class-wc-connect-nux.php

```diff
@@ -156,7 +156,30 @@ public function register_order_page_labels_pointer( $pointers ) {
 			return $pointers;
 		}
 
-		public static function get_banner_type_to_display( $status = array() ) {
+		/**
+		 * Check that the current user is the owner of the Jetpack connection
+		 * - Only that person can accept the TOS
+		 *
+		 * @uses self::get_jetpack_install_status()
+		 *
+		 * @return bool
+		 */
+		public function can_accept_tos() {
+			// Developer case
+			if ( self::JETPACK_DEV === $this->get_jetpack_install_status() ) {
+				return true;
+			}
+
+			$user_token = Jetpack_Data::get_access_token( JETPACK_MASTER_USER );
+			$can_accept = (
+				isset( $user_token->external_user_id ) &&
+				get_current_user_id() === $user_token->external_user_id
+			);
+
+			return $can_accept;
+		}
+
+		public function get_banner_type_to_display( $status = array() ) {
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-services/pull/1062/commits/0e627381b6a8631509c0c6c785fc78dad2bb863c